pkg/antctl/command_definition.go

```diff
@@ -62,7 +69,7 @@ var groupCommands = map[commandGroup]*cobra.Command{
 }
 
 type endpointResponder interface {
-	single() bool
+	output() OutputType
```

Comment:
IMO, "output" sounds like a noun which is shorter like the original "single". Also, if changed, we will have outputType() as a function and a outputType as a field in nonResourceEndpoint, which is confusing I think.

Reply:
Of course it can sound like a noun but it means the output of the responder? I don't know if it's only me who will be confused by the function name. AFAIK, Output is widely used by exec, command, http request to get the output and the noun output is used to take the return value.
Besides, if output is really clear for the mean of outputType, why the type is named OutputType while the method is named output?
Defining a method to get the field with same name is quite common regardless of languages, it's even encouraged to do it in Golang (to save the prefix Get), I don't think it's confusing.
> If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner.

https://golang.org/doc/effective_go.html#Getters

Reply:
Thanks for sharing the doc. But `outputType()` cannot work, it returns `type nonResourceEndpoint has both field and method named outputType`. I choose `OutputType()` instead.